.github/ISSUE_TEMPLATE/site.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Site improvement
+about: Suggest an improvement to nix.dev (fix typos, styling, etc)
+title: ''
+labels: site
+assignees: ''
+
+---
+
+**Suggestion**
+<!-- What would you like to see changed? -->
+
+**Willing to help?**
+<!-- Are you willing to help make the change? -->
```

Comment:
Every issue should have the 👍 note and a query to issues/PRs sorted by count. We decided a while ago to use this consistently